### ISSUE TITLE
StackOverflowError at getStringSet() call

### ISSUE THREAD

**Issue by wrbl606** [akshay2211/Store#3]
```
java.lang.StackOverflowError: stack size 1037KB
at com.fxn.stash.Stash.getStringSet(Stash.java:128)
at com.fxn.stash.Stash.getStringSet(Stash.java:130)
at com.fxn.stash.Stash.getStringSet(Stash.java:130)
at com.fxn.stash.Stash.getStringSet(Stash.java:130)
```

**Comment by wrbl606** [comment]
I'm unable to get string set with the following code:
```java
    private String name;
    private Set<String> elements;

    public Sample(String name) {
        this.name= name;
        elements = Stash.getStringSet(name, new HashSet<>());
    }
```
Even attempt below does not work:
```java
    private String name;
    private Set<String> elements;

    public Sample(String name) {
        this.name= name;
        elements = Stash.getStringSet(name);

        if (elements == null) elements = new HashSet<>();
    }
```

In both cases the string set does not exist yet.

**Comment by akshay2211** [comment]
Hello @wrbl606,
Thank you for trying out Stash.
This bug is been fixed in the latest release [1.3.2](https://github.com/akshay2211/Stash#download)
